affinityworks/main
# resolve "member landing page" and "show correct join link on dashboard"

resolves #620 
resolves #637 
____________________________________

# Notes

* upon joining a group, a member is logged into their homepage
* they see a group name and if they click on it, they see a stripped-down version of the group dashboard
* that page includes a link to the *correct* join page (as well as google group links, if enabled)
* for testing/demo: add a new `No Events Group` to the dev dataset and introduce a new post-deploy hook to make sure this (and all networks/groups in `networks.yml` are present at deploy-time in all apps)
* note: i just got rid of the entire nav tab, since we specified to not show all tabs except "dashboard" which is already visible in the breadcrumb. (easy enough to bring it back later if we need it)

# Testing Demo Guide

to test this:
* log in as `organizer@member.com`
* click on the `No Events Group` link
* click on the join form link on the dashboard
* join as a new member
* click on the `No Events Group` link
* you should see a dashboard without organizer features

to test this with google group links:
* log in as `organizer@member.com`
* visit groups/63/subgroups/new (the new subgroup page for the `No Events Group`)
* make a new group, wait to get redirected to dashboard
* click on the join form link on the dashboard
* join as a new member
* click on the link to the group you just made
* you should see a dashboard without organizer features (and google group link)

# Screenshots

*member view of dashboard:*

![dashboard-member-view](https://user-images.githubusercontent.com/6032844/39156926-652d5eee-4726-11e8-8811-027a1b4a566f.png)

*organizer view of dashboard:*

![dashboard-organizer-view](https://user-images.githubusercontent.com/6032844/39156945-720ff8a6-4726-11e8-8b19-465a1cccb665.png)
